telemetry/definitions/commonDefinitions.json

```diff
@@ -1545,7 +1555,7 @@
         },
         {
             "name": "aws_experimentActivation",
-            "description": "An experiment was activated or deactivated in the toolkit",
+            "description": "An experiment was activated or deactivated in the Toolkit",
             "metadata": [
                 {
                     "type": "experimentId"
```

Comment:
Thinking we should keep it because it's specific to experiments while still being generic to features. Plus it's already being used, easier to let it be.